src/pybind/mgr/dashboard/frontend/src/index.html

```diff
@@ -3,10 +3,9 @@
 <head>
   <meta charset="utf-8">
   <title>Ceph</title>
-  <base href="/">
 
   <script>
-    window['base-href'] = window.location.pathname;
+    window['_app_base'] = <base href="' + document.location+ '" />;
```

Comment:
As I didn't see in the [HTML `window` spec](https://html.spec.whatwg.org/multipage/window-object.html#dom-document-2) any reference to `_app_base`, I guess it's simply a random string (the same `base-href` would have worked) and that what makes this work is the complete `base` tag instead of the `window.location.pathname`?